_inc/client/components/jetpack-notices/state-notices.jsx

```diff
@@ -235,7 +235,18 @@ class JetpackStateNotices extends React.Component {
 					</NoticeAction>
 				);
 				break;
-
+			case 'reconnection_completed':
+				message = jetpackCreateInterpolateElement(
+					__(
+						'Jetpack successfully reconnected! You can check your Jetpack Connection health by visiting the <a>Site Health tool</a>.',
+						'jetpack'
+					),
+					{
+						a: <a href="site-health.php" />,
```

Comment:
Should we bring `siteAdminUrl` from the props to build that URL?

Reply:
@jeherve  Yes, you are right! Did that!